ctru-rs/examples/audio-filters.rs

```diff
@@ -5,6 +5,7 @@
 #![feature(allocator_api)]
 
 use std::f32::consts::PI;
+use std::ops::Deref;
```

Comment:
Every once in a while we have to thank GitHub for these automated lint messages.